ghcup-0.0.7.yaml

```diff
@@ -2735,6 +2734,90 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~ghc/9.2.6/ghc-9.2.6-aarch64-apple-darwin.tar.xz
               dlSubdir: ghc-9.2.6-aarch64-apple-darwin
               dlHash: 68e54fd1af808395584e73b6a9a5b6c7dd85d451da00429aa124a821f4e96ad0
+    9.2.7:
+      viTags:
+      - base-4.16.4.0
+      - Recommended
```

Comment:
We'll wait for a week before setting `Recommended`. There are no HLS binaries either.